packages/db/postgres/tables.sql

```diff
@@ -209,8 +209,9 @@ CREATE TABLE IF NOT EXISTS psa_pin_request
   -- Points to the pinned content, it is updated once the content is actually being found.
   content_cid     TEXT                                                         NOT NULL REFERENCES content (cid),
   -- The id of the content being requested, it could not exist on IPFS (typo, node offline etc)
-  source_cid   TEXT NOT NULL,
+  source_cid      TEXT NOT NULL,
   name            TEXT,
+  metadata        jsonb,
```

Comment:
Shall we use meta instead for consistency with [nft.storage](https://github.com/nftstorage/nft.storage/blob/main/packages/api/db/tables.sql#L130)?

Reply:
👍